models/DataObject/Service.php

```diff
@@ -217,7 +213,16 @@ public function copyAsChild($target, $source)
 
         if ($new instanceof Concrete) {
             foreach ($new->getClass()->getFieldDefinitions() as $fieldDefinition) {
-                if ($fieldDefinition->getUnique()) {
+                if ($fieldDefinition instanceof DataObject\ClassDefinition\Data\Localizedfields) {
+                    foreach ($fieldDefinition->getFieldDefinitions() as $localizedFieldDefinition) {
+                        if ($fieldDefinition->getUnique()) {
```

Comment:
This needs to be `$localizedFieldDefinition` instead of `$fieldDefinition`.

Reply:
Thanks for testing, fixed it https://github.com/pimcore/pimcore/pull/11214/commits/c1fb7af9d7e4c8cddc5db3f77de5a66d49e22988